lib/torrent.js

```diff
@@ -500,6 +500,12 @@ class Torrent extends EventEmitter {
       this._onWireWithMetadata(wire)
     })
 
+    const onPiecesVerified = (err) => {
```

Comment:
This function is only used in the else case below. Can you move it into that else block?

Reply:
Sure